### ISSUE TITLE
Is there an example of how to use this with an Ember app?

### ISSUE THREAD

**Issue by timothyerwin** [acss-io/atomizer#285]
For example, as an addon or in the pipeline somewhere?

**Comment by tom76kimo** [comment]
Do you use Ember with Webpack? If so, [acss-io/webpack-atomizer-loader](https://github.com/acss-io/webpack-atomizer-loader) can help you.

**Comment by timothyerwin** [comment]
No, I use `ember-cli` and I don't plan to use `webpack`...I was able to get it working by using the Atom plugin which generates a file which I include using SASS.

**Comment by tom76kimo** [comment]
Or you can use `atomizer` cli directly? Since you use `ember-cli` anyway.

**Comment by timothyerwin** [comment]
Yeah, I was just looking for a way to get it to run automatically. Set and forget, like an add-on or part of the build process.

**Comment by tom76kimo** [comment]
Got it. If you could use one of the tools below, would be more possible to achieve your goal.
- [grunt-atomizer](https://github.com/acss-io/grunt-atomizer)
- [gulp-atomizer](https://github.com/acss-io/gulp-atomizer)
- [webpack-atomizer-loader](https://github.com/acss-io/webpack-atomizer-loader)

**Comment by renatoi** [comment]
Ember uses ember-cli (and broccoli if I'm not mistaken) for asset compilation. You can create an ember-cli addon for ACSS. The api is available from atomizer already, see:
https://github.com/acss-io/atomizer#api

More info: https://ember-cli.com/asset-compilation

It should be pretty easy to add it to the ember build.

**Comment by renatoi** [comment]
An example: https://github.com/drewcovi/ember-cli-stylus/
